src/data/wallets/wallet-data.ts

```diff
@@ -782,11 +782,11 @@ export const walletsData: WalletData[] = [
     ],
   },
   {
-    last_updated: "2022-06-24",
+    last_updated: "2024-12-23",
     name: "Pillar",
     image: PillarImage,
-    twBackgroundColor: "bg-[#7501D9]",
-    twGradiantBrandColor: "from-[#7501D9]",
+    twBackgroundColor: "bg-[#6A12D0]",
+    twGradiantBrandColor: "from-[#6A12D0]",
     url: "https://www.pillar.fi/",
     active_development_team: true,
     languages_supported: ["en", "bs", "zh"],
```

Comment:
```suggestion
    languages_supported: ["en"],
```

Reply:
based on the https://github.com/ethereum/ethereum-org-website/issues/14360 I see only english support, @arddluma and @alexandradev can you confirm and modify this

Reply:
hello there, 
Please check this screenshot from pillar wallet
![IMG_1882](https://github.com/user-attachments/assets/89fb72ca-7f9f-4504-8a8e-c70d074bb0b0)

Reply:
i see, so the issue was filled out wrongly?

Reply:
that is correct, apologies for that!

Reply:
done sir, the missing language is Amharic. I have just added it, do you mind checking it please?